tests/test_env.py

```diff
@@ -88,30 +84,22 @@ def test_session_env_lazy(monkeypatch):
     monkeypatch.undo()
 
 
-def test_session_lazy_env_error():
-    """Trying to credentialize an unentered env raises EnvError."""
-    s = Env(aws_access_key_id='id', aws_secret_access_key='key',
-            aws_session_token='token', region_name='null-island-1')
-    with pytest.raises(EnvError):
-        s.get_aws_credentials()
-
-
 @mingdalversion
 @credentials
 def test_s3_open_with_session():
     """Read from S3 using env.open() demonstrating lazy credentials."""
-    with Env() as env:
+    with rasterio.env.Env() as env:
         assert env._creds is None
-        with env.open(L8TIF) as dataset:
+        with rasterio.open(L8TIF) as dataset:
             assert env._creds
             assert dataset.count == 1
 
 
 @mingdalversion
 def test_open_https_vsicurl():
     """Read from https using env.open()."""
-    with Env() as env:
-        with env.open(httpstif) as dataset:
+    with rasterio.env.Env():
+        with rasterio.open(httpstif) as dataset:
```

Comment:
@geowurster @perrygeo I've come full circle, but with lazy AWS credentials, and a better implementation for GDAL configuration 😀